projboard/tests/test_view_article.py

```diff
@@ -36,15 +36,15 @@ def create_articles(self, create_user, create_subjects):
     @pytest.fixture
     @pytest.mark.django_db
     def create_view(self, create_user, create_articles):
-        view = View(user_id=create_user, article_id=create_articles)
-        view.save()
-        return view
+        view_article = View_Article(user_id=create_user, article_id=create_articles)
+        view_article.save()
+        return view_article
 
     @pytest.mark.django_db
     def test_create_view(self, create_view):
-        assert create_view in View.objects.all()
+        assert create_view in View_Article.objects.all()
 
     @pytest.mark.django_db
     def test_amount_of_views_article(self, create_view):
-        amount_views = View.amount_of_views_article(create_view.article_id)
-        assert len(View.objects.filter(article_id=create_view.article_id)) == amount_views
+        amount_views = View_Article.amount_of_views_article(create_view.article_id)
+        assert len(View_Article.objects.filter(article_id=create_view.article_id)) == amount_views
```

Comment:
could use Count function instead of len.

Reply:
You right, but this PR just renaming the View model, I didn't changed any method.
Please resolve this conversation.

Reply:
@Shalevro2 @Anas-mhameed 
You could agree of opening new issue mention it here
And continue with this Review

Reply:
@Yarboa 
Opened issue #53 for it

Reply:
@finziamit Please feel free to submit a fix also :) it is simple